conans/test/compile_helpers_test.py

```diff
@@ -97,7 +97,16 @@ def configure_environment_test(self):
                                             '"path/to/includes/lib2"')
         # Not supported yet
         env = ConfigureEnvironment(BuildInfoMock(), MockWinGccSettings())
-        self.assertEquals(env.command_line, "")
+        self.assertEquals(env.command_line, 'env LIBS="-llib1 -llib2" LDFLAGS="-Lpath/to/lib1 '
```

Comment:
I don't understand this change, how is it possible that it was passing? these are the standard env flags, not really related to this feature